docs/en/ingest-management/fleet/enrollment-handling-containerized-agent.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[containers-agent-enrollment-handling]]
+= Enrollment handing for containerized {fleet}-managed agents
+++++
+<titleabbrev>Enrollment handing for containerized agents</titleabbrev>
+++++
+
+Beginning with version 8.18, for {fleet}-managed {agents} that run in a containerized environment (including Docker, Kubernetes, and others), enrollment handling is managed as follows:
+
+* **Enrollment Verification:** The agent checks stored enrollment conditions stored in the container environment and re-enrolls only when necessary.
+* **Unenrollment Handling:** If an {agent} is unenrolled through the {fleet} UI but still has a valid enrollment token provided via environment variables, the next container restart will cause it to re-enroll automatically.
```

Comment:
```suggestion
* **Enrollment Verification:** The {agent} checks the stored enrollment conditions within its container environment and re-enrolls only when necessary.
* **Unenrollment Handling:** If an {agent} is unenrolled via the {fleet} UI but still references a valid enrollment token provided through environment variables, it will automatically re-enroll on the next container restart.
```

Reply:
Looks good. Thanks! I missed that doubled "stored". :-)